UnityProject/Assets/Scripts/Effects/LTEffect.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using Mirror;
+
+public class LTEffect : NetworkBehaviour 
+{
+    /// <summary>
+    /// The base class for all animated effects that requires LeanTween to sync between all clients.
+    /// </summary>
+
+    [HideInInspector]
+    public Vector3 originalPosition;
+
+    [Tooltip("Which Axis will the animation play on?")]
+    public NetworkedLeanTween.Axis axisMode = NetworkedLeanTween.Axis.X;
```

Comment:
whenever possible, prefer to use ``private`` and ``[SerializedField]``

Reply:
we could need to change the axisMode of the component via script in the future so it's best to leave it public.

Reply:
alright, that's fine. I'm just seeing a lot of publics in your code when I feel it isn't needed. Use publics sparingly and only when you intend it to be accessed and modified from an arbitrary script, otherwise always restrict the access to read only or make setters.